ocs_ci/utility/rosa.py

```diff
@@ -47,11 +47,21 @@ def create_cluster(cluster_name, version):
     compute_nodes = config.ENV_DATA["worker_replicas"]
     compute_machine_type = config.ENV_DATA["worker_instance_type"]
     multi_az = "--multi-az " if config.ENV_DATA.get("multi_availability_zones") else ""
+    cluster_type = config.ENV_DATA.get("cluster_type", "")
+    provider_name = config.ENV_DATA.get("provider_name", "")
     cmd = (
         f"rosa create cluster --cluster-name {cluster_name} --region {region} "
         f"--compute-nodes {compute_nodes} --mode auto --compute-machine-type "
         f"{compute_machine_type}  --version {rosa_ocp_version} {multi_az}--sts --yes"
     )
+    if cluster_type.lower() == "consumer" and config.ENV_DATA.get("provider_name", ""):
+        subnet_id = get_providers_subnet(region, provider_name)
+        cmd = (
+            f"rosa create cluster --cluster-name {cluster_name} --region {region} "
+            f"--compute-nodes {compute_nodes} --mode auto --compute-machine-type "
+            f"{compute_machine_type}  --version {rosa_ocp_version} --subnet-ids {subnet_id} {multi_az}--sts --yes"
```

Comment:
The command looks similar to command on line 52. Could it be reused by only appending ` --subnet-ids {subnet_id}` so that we don't manage the command on 2 places if it changes?

Reply:
Cheanged